package.json

```diff
@@ -21,6 +21,7 @@
   },
   "homepage": "https://github.com/distributedlab-solidity-library/hardhat-gobind#readme",
   "devDependencies": {
```

Comment:
We have to talk about the required dependencies. What is used right now is not enough